file_example.c

```diff
@@ -193,13 +194,26 @@ static int file_flush(struct tcmu_device *dev, struct tcmulib_cmd *cmd)
 	return 0;
 }
 
+static int file_reconfig(struct tcmu_device *dev, struct tcmulib_cfg_info *cfg)
+{
+	switch (cfg->type) {
+	case TCMULIB_CFG_DEV_SIZE:
+		return tcmu_update_num_lbas(dev, cfg->data.dev_size);
```

Comment:
[0001-tcmu-Add-UA-to-reconfig.txt](https://github.com/open-iscsi/tcmu-runner/files/1103725/0001-tcmu-Add-UA-to-reconfig.txt)